app/src/main/java/valentyn/androidtasks/graphics/ColorPickerDialog.kt

```diff
@@ -0,0 +1,221 @@
+package valentyn.androidtasks.graphics
+
+import android.os.Bundle
+import android.app.Dialog
+import android.content.Context
+import android.graphics.*
+import android.view.MotionEvent
+import android.view.View
+
+class ColorPickerDialog(
+    context: Context,
+    private val listener: OnColorChangedListener,
+    private val initialColor: Int
+) : Dialog(context) {
```

Comment:
You should set all parameters to the Dialog(context, attr, deff), other way you will have a problems with xml.
Read about @JvmOverload